cmd/protoc-gen-loggingtags/README.md

```diff
@@ -0,0 +1,54 @@
+# protoc-gen-loggingtags
```

Comment:
Do we want this in heroku/x or its own repo?

Reply:
I think in here is fine, unless it changes some of the answers to the Makefile questions.